src/ucp/rndv/rndv.c

```diff
@@ -656,6 +657,13 @@ static void ucp_rndv_req_init_zcopy_lane_map(ucp_request_t *rndv_req,
             continue;
         }
 
+        /* Ignore GDR lane if length is greater than pipeline rndv threshold */
+        if ((mem_type != UCS_MEMORY_TYPE_HOST) &&
+            (md_attr->cap.reg_mem_types & UCS_BIT(UCS_MEMORY_TYPE_HOST)) &
+            (length > context->config.ext.rndv_pipeline_thresh)) {
+                continue;
+        }
```

Comment:
1. fix indent
2. can we do this earlier during RTS stage? to avoid extra checks